helpers/ShareArticle.js

```diff
@@ -0,0 +1,73 @@
+import Debug from 'debug';
+import dotenv from 'dotenv';
+import Postals from './Postals';
+
+const debug = Debug('dev');
+dotenv.config();
+/**
+ *
+ *
+ * @export
+ * @class EmailVerification
+ */
+export default class ShareArticle {
+  /**
+   *
+   * @static
+   * @param {string} requestInfo - request information
+   * @param {string} recipient - recipient of the email
+   * @param {string} title - title of the article
+   * @param {string} slug - article slug
+   * @memberof ShareArticleByMail
+   * @returns {function} - returns a function call
+   */
+  static shareArticleByMail(requestInfo, recipient, title, slug) {
+    const { firstName, lastName, id } = requestInfo.user;
+    const url = `${requestInfo.protocol}://${requestInfo.get('host')}/api/v1/articles/${id}/${slug}`;
```

Comment:
This was the point I was making in the morning. The `url` of the article you share is not dynamic, it wall always point to the API.

What then happens when we have a frontend and a different article URL?